lighting/views/views.xml

```diff
@@ -196,7 +196,6 @@
                         </page>
                         <page string="Lighting">
                             <group>
-                                <field name="total_nominal_flux"/>
```

Comment:
You also need to remove the field "total_nominal_flux" from lighting.product https://github.com/nuobit/lighting-vertical/blob/c7ae2fcf6049cea36c232dcb10ce4d72942f61a4/lighting/models/product.py#L637